html.md

```diff
@@ -18,28 +18,29 @@ Most tags also have a matching closing tag that is written **almost** exactly th
 </tagname>
 ```
 
-Inside of the closing and opening tags, we can place content such as text. The full group of opening tag, content and a closing tag is what is known as an _element_.
+Inside of the closing and opening tags, we can place content such as text.
```

Comment:
?? It's called an element. We're going to be calling them elements, so why not explain the difference?